### PR TITLE
Add method to do misc JSONRPC calls

### PR DESCRIPTION
**Pull request by jnordberg**
Needed for condenser now that we will start calling other services through steem-js

### CODE REVIEW

File: src/api/index.js [hunk 1]
```diff
@@ -117,6 +121,15 @@ class Steem extends EventEmitter {
         return this.transport.send(api, data, cb);
     }
 
+    call(method, params, callback) {
```
Comment (relativityboy) [inline src/api/index.js]: I suggest that call() be renamed rpc()

File: src/api/transports/http.js [hunk 2]
```diff
@@ -6,31 +6,44 @@ import Transport from './base';
...
+class RPCError extends Error {
+  constructor(rpcError) {
+    super(rpcError.message);
+    this.name = 'RPCError';
+    this.code = rpcError.code;
+    this.data = rpcError.data;
+  }
+}
+
+export function jsonRpcCall(uri, {method, id, params}) {
```
Comment (relativityboy) [inline src/api/transports/http.js]: I suggest jsonRpcCall() -> jsonRpc() || rpc()

This one's not a big deal, but the shorter names are dryer and a little more elegant.

File: src/api/index.js (hunk 1 above)
Comment (bonustrack) [reply src/api/index.js]: Just wondering why do we need this function? There is already `steem.api.send()` which give similar result and not specific to a transport.

Comment (jnordberg) [reply src/api/index.js]: We are moving all services to jussi which will be a proxy for both steemd and our services and it uses jsonrpc differently (the rpc method isn't hard-coded to `call`) so I needed to separate it. We will also be adding request signing to this at some point.

My thinking is that we use `.call` for misc json rpc service calls for now and when we are ready to drop websocket entirely we route everything through the `.call` method and remove `.send`.

Re, naming I think `steem.api.call()` makes more sense than `steem.api.rpc()`

File: src/api/transports/http.js (hunk 2 above)
Comment (jnordberg) [reply src/api/transports/http.js]: Noted :) RIP in peace

**Pull request merged**